Kickstarter-iOS/Views/Cells/PillCell.swift

```diff
@@ -34,6 +34,7 @@ final class PillCell: UICollectionViewCell, ValueCell {
 
     _ = self.contentView
       |> contentViewStyle
+      |> \.backgroundColor .~ .ksr_grey_200
```

Comment:
Is this right? Shouldn't the background be green?

![image](https://user-images.githubusercontent.com/3156796/63043045-595c3680-be99-11e9-9955-81d0498407d8.png)

Reply:
Apologies 🤦‍♂